packages/next/src/utils.ts

```diff
@@ -2506,7 +2547,7 @@ export async function getMiddlewareBundle({
                   ...wasmFiles,
                   ...assetFiles,
                 },
-                regions: edgeFunction.regions,
+                regions: normalizeRegions(edgeFunction.regions ?? []),
```

Comment:
Does an empty array behave the same as `undefined`? If so seems good

Reply:
Changed it to be `undefined` when there are no regions